### PR TITLE
Add new feature of servicemesh

### PR DESCRIPTION
**Pull request by liuzhiyi1993**
Implementing EdgeMesh

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind feature

**What this PR does / why we need it**:
>
>Add new feature of servicemesh
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: edge/pkg/metamanager/client/endpoint.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
```
Comment (sids-b) [inline edge/pkg/metamanager/client/endpoint.go]: please arrange the imports.
Applicable to similar places

File: edgemesh/cmd/conf/chassis.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+ssl:
```
Comment (sids-b) [inline edgemesh/cmd/conf/chassis.yaml]: please add a blank line at the end of the file.
Applicable to similar places

File: edgemesh/pkg/module.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+import (
+	"github.com/kubeedge/beehive/pkg/core"
+	"github.com/kubeedge/beehive/pkg/core/context"
+	"github.com/kubeedge/kubeedge/edge/pkg/common/modules"
+	"github.com/kubeedge/kubeedge/edgemesh/pkg/constant"
+	"github.com/kubeedge/kubeedge/edgemesh/pkg/server"
+	"github.com/kubeedge/beehive/pkg/common/log"
+
+	//_ "github.com/kubeedge/kubeedge/edgemesh/pkg/registry"
```
Comment (sids-b) [inline edgemesh/pkg/module.go]: please remove commented lines

File: edgemesh/pkg/server/tcp.go [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+		respString = respString[0 : len(respString) - 2]
+		respString += "\n"
+	}
+	b, _  := ioutil.ReadAll(resp.Body)
+	respString += "\n" + string(b)
+	return respString
+}
+
+func process(conn net.Conn) {
+	fmt.Println("start receiving data...")
```
Comment (sids-b) [inline edgemesh/pkg/server/tcp.go]: please remove print statements.... use log instead

**Comment by sids-b** [comment]
please add edgemesh in conf/modules.go

File: edge/pkg/metamanager/client/endpoint.go (hunk 1 above)
Comment (liuzhiyi1993) [reply edge/pkg/metamanager/client/endpoint.go]: done

File: edgemesh/pkg/module.go (hunk 3 above)
Comment (liuzhiyi1993) [reply edgemesh/pkg/module.go]: done

File: edgemesh/cmd/conf/chassis.yaml (hunk 2 above)
Comment (liuzhiyi1993) [reply edgemesh/cmd/conf/chassis.yaml]: done

File: edgemesh/pkg/server/tcp.go (hunk 4 above)
Comment (liuzhiyi1993) [reply edgemesh/pkg/server/tcp.go]: done

**Comment by kevin-wangzefeng** [comment]
LGTM, thanks.
Manual merging to squash the commits.

**Pull request merged**